### DISCUSSION TITLE
iOS App in Public Beta 🎉

### DISCUSSION THREAD

**Discussion by viktorstrate** [photoview/photoview#476] [Show and tell]
Hi everyone, I'm happy to announce that the first beta version of the new iOS app has just been approved by Apple and is now in public beta.

I'd much appreciate it if you considered joining the beta and helped provide feedback, either through TestFlight, or in this thread.
You can join here: https://testflight.apple.com/join/x66dDuYX

When logging in, add `/api` at the end of your instance URL.
If you don't have your own server, you can also use the public demo account by using the following credentials:

Instance URL: `https://photos.qpqp.dk/api`
Username: `demo`
Password: `demo`

**Comment by FuJuntao** [comment]
Hi, I have a few questions about the iOS app
Does it support HEIC?
Dose is support Live Photo?
Dose is support Portrait Mode?
If I edit the photo, and upload it to the server, and download it again, will it be able to be revert the editing?
